### ISSUE TITLE
Meta Quest Browser Support

### ISSUE THREAD

**Issue by forEachWhileTrue** [immersive-web/WebXR-WebGPU-Binding#14]
Hello everyone,

I'm currently researching in the direction of human performance capture with dynamic gaussian splatting and want to implement a performant web viewer which runs on Meta Quest 2 and 3.

Therefore I would love to use the WebXR-WebGPU-Bindings but the [WebXR report page](https://immersive-web.github.io/webxr-samples/report/) indicates that these bindings are not supported when I check it on my Meta Quest Browser with the latest update.

My Problem is, that I'm already in development and need this feature as soon as possible or I have to work with WebGL which does not support compute shaders. So my questions is, do you have any information at all on when the bindings will support which platform?

Thanks in advance!

**Comment by jpryne** [comment]
Yes, six months later I'm seeing this too. What remains to be done to get this incorporated and allow developers to utilize webgpu and webxr together?
